code/modules/admin/chat_commands.dm

```diff
@@ -78,13 +78,13 @@ GLOBAL_LIST(round_end_notifiees)
 /datum/tgs_chat_command/notify
 	name = "notify"
 	help_text = "Pings the invoker when the round ends"
-	admin_only = TRUE
+	admin_only = FALSE
 
 /datum/tgs_chat_command/notify/Run(datum/tgs_chat_user/sender, params)
 	if(!SSticker.IsRoundInProgress() && SSticker.HasRoundStarted())
 		return "[sender.mention], the round has already ended!"
 	LAZYINITLIST(GLOB.round_end_notifiees)
-	GLOB.round_end_notifiees[sender.mention] = TRUE
+	GLOB.round_end_notifiees["<@[sender.mention]>"] = TRUE

```

Comment:
You know this will paste into the discord <@<@200631029675982858>>, Right @kevinz000? `sender.mention` includes the mention format system `<@ID>`. See https://github.com/tgstation/tgstation/pull/44231 for a better implementation which pings on full restart, and has sanitzation procs.

Reply:
https://www.youtube.com/watch?v=SGDpuxuBByE